### PR TITLE
TRUNK-5134: Replace path constructions using separator char

### PR DESCRIPTION
**Pull request by wikumChamith**
## Description of what I changed

Occurrences, where file paths are constructed using "file.separator", is refactored by using Paths.

## Issue I worked on

see https://issues.openmrs.org/browse/TRUNK-5134

## Checklist: I completed these to help reviewers :)

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

File: api/src/main/java/org/openmrs/liquibase/ChangeLogVersionFinder.java [hunk 1]
```diff
@@ -28,15 +29,13 @@
...
-	static final String BASE_FOLDER_NAME = "org" + File.separator + "openmrs" + File.separator + "liquibase";
+	static final String BASE_FOLDER_NAME = Paths.get("org",  "openmrs", "liquibase").toString();
 	
-	static final String CORE_DATA_FOLDER_NAME = BASE_FOLDER_NAME + File.separator + "snapshots" + File.separator
-	        + "core-data";
+	static final String CORE_DATA_FOLDER_NAME = Paths.get(BASE_FOLDER_NAME, "snapshots", "core-data").toString();
 	
-	static final String SCHEMA_ONLY_FOLDER_NAME = BASE_FOLDER_NAME + File.separator + "snapshots" + File.separator
-	        + "schema-only";
+	static final String SCHEMA_ONLY_FOLDER_NAME = Paths.get(BASE_FOLDER_NAME,"snapshots", "schema-only").toString();
```
Comment (tendomart) [inline api/src/main/java/org/openmrs/liquibase/ChangeLogVersionFinder.java:36]: Was spacing necessary here  `BASE_FOLDER_NAME,"snapshots"`

File: api/src/main/java/org/openmrs/liquibase/ChangeLogVersionFinder.java [hunk 2]
```diff
@@ -98,8 +97,8 @@ public SortedMap<String, List<String>> getSnapshotCombinations() {
 	
 	public List<String> getSnapshotFilenames(String version) {
 		String versionAsDotX = getVersionAsDotX(version);
-		return Arrays.asList(SCHEMA_ONLY_FOLDER_NAME + File.separator + SCHEMA_ONLY_BASE_NAME + versionAsDotX + DOT_XML,
-		    CORE_DATA_FOLDER_NAME + File.separator + CORE_DATA_BASE_NAME + versionAsDotX + DOT_XML);
+		return Arrays.asList(Paths.get(SCHEMA_ONLY_FOLDER_NAME,SCHEMA_ONLY_BASE_NAME + versionAsDotX + DOT_XML).toString(),
```
Comment (tendomart) [inline api/src/main/java/org/openmrs/liquibase/ChangeLogVersionFinder.java:100]: and here` Paths.get(SCHEMA_ONLY_FOLDER_NAME,SCHEMA_ONLY_BASE_NAME`

File: api/src/main/java/org/openmrs/liquibase/ChangeLogVersionFinder.java [hunk 3]
```diff
@@ -108,12 +107,12 @@ public Optional<String> getLatestSnapshotVersion() {
 	
 	public Optional<String> getLatestSchemaSnapshotFilename() {
 		return getLatestSnapshotVersion().map(
-			snapshotVersion -> SCHEMA_ONLY_FOLDER_NAME + File.separator + SCHEMA_ONLY_BASE_NAME + snapshotVersion + DOT_XML);
+			snapshotVersion -> Paths.get(SCHEMA_ONLY_FOLDER_NAME ,SCHEMA_ONLY_BASE_NAME + snapshotVersion + DOT_XML).toString());
```
Comment (tendomart) [inline api/src/main/java/org/openmrs/liquibase/ChangeLogVersionFinder.java:110]: and here `Paths.get(SCHEMA_ONLY_FOLDER_NAME ,SCHEMA_ONLY_BASE_NAME +`

File: api/src/main/java/org/openmrs/liquibase/ChangeLogVersionFinder.java [hunk 4]
```diff
@@ -108,12 +107,12 @@ public Optional<String> getLatestSnapshotVersion() {
...
 	}
 	
 	public Optional<String> getLatestCoreDataSnapshotFilename() {
 		return getLatestSnapshotVersion().map(
-			snapshotVersion -> CORE_DATA_FOLDER_NAME + File.separator + CORE_DATA_BASE_NAME + snapshotVersion + DOT_XML);
+			snapshotVersion -> Paths.get(CORE_DATA_FOLDER_NAME,CORE_DATA_BASE_NAME + snapshotVersion + DOT_XML).toString());
```
Comment (tendomart) [inline api/src/main/java/org/openmrs/liquibase/ChangeLogVersionFinder.java:115]: same here `Paths.get(CORE_DATA_FOLDER_NAME,CORE_DATA_BASE_NAME`

File: api/src/main/java/org/openmrs/liquibase/ChangeLogVersionFinder.java (hunk 1 above)
Comment (wikumChamith) [reply api/src/main/java/org/openmrs/liquibase/ChangeLogVersionFinder.java:36]: Spacing is present in other instances.

https://github.com/openmrs/openmrs-core/blob/d5d65c823890ce533d70d5e308eaccf59d3478a4/liquibase/src/main/java/org/openmrs/liquibase/Main.java#L19-L29

Comment (tendomart) [reply api/src/main/java/org/openmrs/liquibase/ChangeLogVersionFinder.java]: okk

**Comment by wikumChamith** [comment]
@dkayiwa Please review this.

**Pull request closed without merging**